components.json

```diff
@@ -6,12 +6,12 @@
   "tailwind": {
     "config": "tailwind.config.js",
     "css": "src/index.css",
-    "baseColor": "neutral",
-    "cssVariables": true
+    "baseColor": "gray",
+    "cssVariables": true,
+    "prefix": "yes"
   },
   "aliases": {
     "components": "@/components/shadcn-ui",
-    "utils": "@/lib/utils",
-    "ui": "@/components/shadcn-ui"
```

Comment:
These are weird, can you undo your changes on components.json and move the stuffs you just generated to
src/components/shadcn-ui instead of src/components/shadcn-ui/ui